### DISCUSSION TITLE
"Series" feature

### DISCUSSION THREAD

**Discussion by stereobooster** [jpanther/congo#838] [Ideas]
Inspired by [dev.to](https://dev.to/kallmanation/dev-to-writing-making-a-series-3h79)

Basically, author adds `series` field to frontmatter. All articles with the same `series` considered to be part of multi-part series. We can show series navigation in the right sidebar, for example.

I already implemented this feature once in Hugo - I can contribute if you want.

PS. I know you are busy. And theme is already lit 🔥. Just an idea for you.

**Comment by jpanther** [comment]
I like this idea. Happy to look at a PR if you have time to come up with something. I think linking series in the sidebar is a good option, and it could potentially override the article navigation links at the bottom of the article when it's a post that's in a series?

**Reply by stereobooster** [reply]
basic code looks like this (memo for myself)

```liquid
        {{ if .Params.series }}
          <section>
            <p>Part of "{{ .Params.series }}" series</p>
            <ol>
            {{ range (where .Site.Pages ".Params.series" "==" .Params.series).Reverse }}
              <li>
                {{ if eq .Permalink $.Permalink }}
                  {{ .Title }}
                {{ else }}
                  <a href="{{ .Permalink }}">{{ .Title }}</a>
                {{ end }}
              </li>
            {{ end }}
            </ol>
          </section>
        {{ end }}
```
